tox.ini

```diff
@@ -32,11 +32,10 @@ commands = {envpython} -m pylint {posargs:cloudinit tests tools}
 [testenv:py3]
 basepython = python3
 deps =
-    nose-timer
     -r{toxinidir}/test-requirements.txt
-commands = {envpython} -m nose --with-timer --timer-top-n 10 \
-           {posargs:--with-coverage --cover-erase --cover-branches \
-            --cover-inclusive --cover-package=cloudinit \
+commands = pytest \
+            --durations 10 \
+            {posargs:--cov cloudinit --cov-branch \
```

Comment:
Can you change this to `--cov=cloudinit` ?  It makes it obvious that cloud-init is a value to the `--cov` key, rather than some other thing.